[2024-01-06 10:45:41,717: INFO: main: Welecome to our custom log]
[2024-01-06 21:20:21,963: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-06 21:33:35,334: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-06 21:33:35,340: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-06 21:33:35,343: INFO: common: created directory at: artifacts]
[2024-01-06 21:33:35,447: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-06 21:33:35,449: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-06 21:34:15,891: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-07 22:07:21,479: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-07 22:07:21,688: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-07 22:07:21,718: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-07 22:07:21,718: INFO: common: created directory at: artifacts]
[2024-01-07 22:07:21,718: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-07 22:07:21,724: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-07 22:08:07,830: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-07 22:08:08,530: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-08 19:26:52,462: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-08 19:26:52,473: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-08 19:26:52,480: INFO: common: created directory at: artifacts]
[2024-01-08 19:26:52,484: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-08 19:28:08,360: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-09 17:13:20,876: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-09 17:13:20,985: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-09 17:13:21,001: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-09 17:13:21,024: INFO: common: created directory at: artifacts]
[2024-01-09 17:13:21,024: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-09 17:13:21,024: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-09 17:13:55,825: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-09 17:13:56,813: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-09 17:13:56,813: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-09 17:13:56,829: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-09 17:13:56,834: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-09 17:13:56,835: INFO: common: created directory at: artifacts]
[2024-01-09 17:13:56,836: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-09 17:13:57,392: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-09 17:13:58,523: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-09 17:44:00,703: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-09 17:44:00,709: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-09 17:44:00,710: INFO: common: created directory at: artifacts]
[2024-01-09 17:44:00,710: INFO: common: created directory at: artifacts\training]
[2024-01-09 21:58:15,739: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-09 21:58:15,739: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-09 21:58:15,747: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-09 21:58:15,747: INFO: common: created directory at: artifacts]
[2024-01-09 21:58:15,747: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-09 21:58:15,747: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-09 21:59:04,726: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-09 21:59:05,365: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-09 21:59:05,365: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-09 21:59:05,373: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-09 21:59:05,373: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-09 21:59:05,373: INFO: common: created directory at: artifacts]
[2024-01-09 21:59:05,373: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-09 21:59:05,991: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-09 21:59:07,080: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-09 21:59:07,080: INFO: main: >>>>>>> stage Training started]
[2024-01-09 21:59:07,088: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-09 21:59:07,105: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-09 21:59:07,107: INFO: common: created directory at: artifacts]
[2024-01-09 21:59:07,108: INFO: common: created directory at: artifacts\training]
[2024-01-09 22:00:30,506: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-11 20:54:21,920: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-11 20:54:22,320: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-11 20:54:22,324: INFO: common: created directory at: artifacts]
[2024-01-11 20:54:46,900: INFO: common: json file saved at: scores.json]
[2024-01-11 20:54:52,717: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-11 20:54:53,742: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpwiabdn19\model\data\model\assets]
[2024-01-11 21:05:19,737: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-11 21:05:19,742: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-11 21:05:19,744: INFO: common: created directory at: artifacts]
[2024-01-11 21:05:42,482: INFO: common: json file saved at: scores.json]
[2024-01-11 21:06:01,851: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)': /rathorkapil18/Kidney-Disease-Classification-Deep-Learning-Project.mlflow/api/2.0/mlflow/runs/create]
[2024-01-11 21:06:06,294: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-11 21:06:07,305: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpu8cqjyoj\model\data\model\assets]
[2024-01-12 10:35:29,121: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 10:35:29,149: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 10:35:29,151: INFO: common: created directory at: artifacts]
[2024-01-12 10:35:47,324: INFO: common: json file saved at: scores.json]
[2024-01-12 10:35:51,251: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 10:35:51,925: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp_3tvfkir\model\data\model\assets]
[2024-01-12 10:38:04,807: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 10:38:04,814: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 10:38:04,816: INFO: common: created directory at: artifacts]
[2024-01-12 10:38:21,666: INFO: common: json file saved at: scores.json]
[2024-01-12 10:38:24,160: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 10:38:24,781: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpqm3rnj9f\model\data\model\assets]
[2024-01-12 10:39:51,367: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 10:39:51,380: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 10:39:51,380: INFO: common: created directory at: artifacts]
[2024-01-12 10:40:08,592: INFO: common: json file saved at: scores.json]
[2024-01-12 10:40:11,152: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 10:40:11,754: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpd1uvnaam\model\data\model\assets]
[2024-01-12 12:09:55,745: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 12:09:55,751: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:09:55,759: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:09:55,759: INFO: common: created directory at: artifacts]
[2024-01-12 12:09:55,759: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 12:09:55,762: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:10:15,464: ERROR: main: ('Connection aborted.', TimeoutError(10060, 'A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond', None, 10060, None))]
Traceback (most recent call last):
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 491, in _make_request
    raise new_e
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 467, in _make_request
    self._validate_conn(conn)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 1096, in _validate_conn
    conn.connect()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connection.py", line 642, in connect
    sock_and_verified = _ssl_wrap_socket_and_match_hostname(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connection.py", line 782, in _ssl_wrap_socket_and_match_hostname
    ssl_sock = ssl_wrap_socket(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\util\ssl_.py", line 470, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(sock, context, tls_in_tls, server_hostname)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\util\ssl_.py", line 514, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\ssl.py", line 1073, in _create
    self.do_handshake()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
TimeoutError: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\requests\adapters.py", line 486, in send
    resp = conn.urlopen(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 844, in urlopen
    retries = retries.increment(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\util\retry.py", line 470, in increment
    raise reraise(type(error), error, _stacktrace)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\util\util.py", line 38, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 491, in _make_request
    raise new_e
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 467, in _make_request
    self._validate_conn(conn)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connectionpool.py", line 1096, in _validate_conn
    conn.connect()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connection.py", line 642, in connect
    sock_and_verified = _ssl_wrap_socket_and_match_hostname(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\connection.py", line 782, in _ssl_wrap_socket_and_match_hostname
    ssl_sock = ssl_wrap_socket(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\util\ssl_.py", line 470, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(sock, context, tls_in_tls, server_hostname)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\urllib3\util\ssl_.py", line 514, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\ssl.py", line 1073, in _create
    self.do_handshake()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', TimeoutError(10060, 'A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond', None, 10060, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.download_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 31, in download_file
    raise e
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 26, in download_file
    gdown.download(prefix+file_id,zip_download_dir)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 161, in download
    res = sess.get(url, stream=True, verify=verify)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\requests\sessions.py", line 602, in get
    return self.request("GET", url, **kwargs)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\requests\sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\requests\sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\requests\adapters.py", line 501, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', TimeoutError(10060, 'A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond', None, 10060, None))
[2024-01-12 12:12:20,115: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 12:12:20,117: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:12:20,117: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:12:20,117: INFO: common: created directory at: artifacts]
[2024-01-12 12:12:20,117: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 12:12:20,117: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:12:23,668: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:12:23,668: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 12:18:36,108: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 12:18:36,111: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:18:36,114: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:18:36,114: INFO: common: created directory at: artifacts]
[2024-01-12 12:18:36,116: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 12:18:36,116: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:18:38,378: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:18:38,378: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 12:49:56,666: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 12:49:56,670: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:49:56,670: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:49:56,670: INFO: common: created directory at: artifacts]
[2024-01-12 12:49:56,670: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 12:49:56,670: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:49:58,864: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:49:58,868: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 12:56:04,480: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 12:56:04,487: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:56:04,502: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:56:04,502: INFO: common: created directory at: artifacts]
[2024-01-12 12:56:04,504: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 12:56:04,504: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:56:05,504: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:56:05,505: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 12:57:32,343: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 12:57:32,346: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:57:32,349: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:57:32,349: INFO: common: created directory at: artifacts]
[2024-01-12 12:57:32,351: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 12:57:32,351: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:57:33,302: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 12:57:34,665: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-12 12:57:34,665: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-12 12:57:34,669: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:57:34,673: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:57:34,673: INFO: common: created directory at: artifacts]
[2024-01-12 12:57:34,673: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-12 12:57:35,303: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-12 12:57:36,664: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-12 12:57:36,670: INFO: main: >>>>>>> stage Training started]
[2024-01-12 12:57:36,670: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:57:36,682: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:57:36,682: INFO: common: created directory at: artifacts]
[2024-01-12 12:57:36,687: INFO: common: created directory at: artifacts\training]
[2024-01-12 12:58:42,556: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-12 12:58:42,558: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-12 12:58:42,561: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 12:58:42,571: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 12:58:42,573: INFO: common: created directory at: artifacts]
[2024-01-12 12:59:02,468: INFO: common: json file saved at: scores.json]
[2024-01-12 12:59:02,468: INFO: common: json file saved at: scores.json]
[2024-01-12 12:59:05,140: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 12:59:05,889: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpauprkgym\model\data\model\assets]
[2024-01-12 12:59:19,471: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\7d212ecc20de4013bb7ae5e2b6290e99\artifacts\model\data]
[2024-01-12 12:59:19,476: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\7d212ecc20de4013bb7ae5e2b6290e99\artifacts\model\data\model]
[2024-01-12 12:59:19,479: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\7d212ecc20de4013bb7ae5e2b6290e99\artifacts\model\data\model\assets]
[2024-01-12 12:59:19,479: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\7d212ecc20de4013bb7ae5e2b6290e99\artifacts\model\data\model\variables]
[2024-01-12 12:59:19,624: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:06:02,926: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 13:06:02,928: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:06:02,930: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:06:02,930: INFO: common: created directory at: artifacts]
[2024-01-12 13:06:02,931: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 13:06:02,931: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:06:03,759: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:06:04,323: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:06:04,323: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-12 13:06:04,338: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:06:04,338: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:06:04,338: INFO: common: created directory at: artifacts]
[2024-01-12 13:06:04,338: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-12 13:06:04,777: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-12 13:06:06,203: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-12 13:06:06,203: INFO: main: >>>>>>> stage Training started]
[2024-01-12 13:06:06,203: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:06:06,203: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:06:06,203: INFO: common: created directory at: artifacts]
[2024-01-12 13:06:06,203: INFO: common: created directory at: artifacts\training]
[2024-01-12 13:06:57,188: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-12 13:06:57,188: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-12 13:06:57,188: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:06:57,202: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:06:57,204: INFO: common: created directory at: artifacts]
[2024-01-12 13:07:13,907: INFO: common: json file saved at: scores.json]
[2024-01-12 13:07:13,907: INFO: common: json file saved at: scores.json]
[2024-01-12 13:07:15,458: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 13:07:16,074: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp1767stl8\model\data\model\assets]
[2024-01-12 13:07:24,398: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\a290d1c3a6d14117b0c775349063ab38\artifacts\model\data]
[2024-01-12 13:07:24,398: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\a290d1c3a6d14117b0c775349063ab38\artifacts\model\data\model]
[2024-01-12 13:07:24,407: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\a290d1c3a6d14117b0c775349063ab38\artifacts\model\data\model\assets]
[2024-01-12 13:07:24,412: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\a290d1c3a6d14117b0c775349063ab38\artifacts\model\data\model\variables]
[2024-01-12 13:07:24,495: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:13:16,602: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 13:13:16,617: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:13:16,617: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:13:16,617: INFO: common: created directory at: artifacts]
[2024-01-12 13:13:16,633: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 13:13:16,633: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:13:17,715: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:13:18,578: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:13:18,578: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-12 13:13:18,581: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:13:18,583: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:13:18,583: INFO: common: created directory at: artifacts]
[2024-01-12 13:13:18,583: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-12 13:13:19,131: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-12 13:13:20,137: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-12 13:13:20,137: INFO: main: >>>>>>> stage Training started]
[2024-01-12 13:13:20,144: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:13:20,146: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:13:20,147: INFO: common: created directory at: artifacts]
[2024-01-12 13:13:20,147: INFO: common: created directory at: artifacts\training]
[2024-01-12 13:15:15,015: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-12 13:15:15,015: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-12 13:15:15,015: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:15:15,031: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:15:15,031: INFO: common: created directory at: artifacts]
[2024-01-12 13:15:33,727: INFO: common: json file saved at: scores.json]
[2024-01-12 13:15:33,727: INFO: common: json file saved at: scores.json]
[2024-01-12 13:15:35,288: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 13:15:35,915: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp4kwkzz41\model\data\model\assets]
[2024-01-12 13:15:43,709: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\01ce3c51626f4d0f8bf044ae403f22e7\artifacts\model\data]
[2024-01-12 13:15:43,709: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\01ce3c51626f4d0f8bf044ae403f22e7\artifacts\model\data\model]
[2024-01-12 13:15:43,709: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\01ce3c51626f4d0f8bf044ae403f22e7\artifacts\model\data\model\assets]
[2024-01-12 13:15:43,725: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\01ce3c51626f4d0f8bf044ae403f22e7\artifacts\model\data\model\variables]
[2024-01-12 13:15:43,813: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:19:51,716: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 13:19:51,716: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:19:51,716: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:19:51,716: INFO: common: created directory at: artifacts]
[2024-01-12 13:19:51,716: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 13:19:51,716: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:19:52,534: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:19:52,941: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:19:52,941: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-12 13:19:52,941: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:19:52,941: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:19:52,941: INFO: common: created directory at: artifacts]
[2024-01-12 13:19:52,957: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-12 13:19:53,376: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-12 13:19:53,803: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-12 13:19:53,803: INFO: main: >>>>>>> stage Training started]
[2024-01-12 13:19:53,803: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:19:53,803: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:19:53,803: INFO: common: created directory at: artifacts]
[2024-01-12 13:19:53,803: INFO: common: created directory at: artifacts\training]
[2024-01-12 13:21:43,382: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-12 13:21:43,382: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-12 13:21:43,391: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:21:43,395: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:21:43,396: INFO: common: created directory at: artifacts]
[2024-01-12 13:22:02,337: INFO: common: json file saved at: scores.json]
[2024-01-12 13:22:02,348: INFO: common: json file saved at: scores.json]
[2024-01-12 13:22:04,088: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 13:22:04,743: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp22akwor2\model\data\model\assets]
[2024-01-12 13:22:12,337: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\984f8177e4e24cb287e1dbd543f5b8be\artifacts\model\data]
[2024-01-12 13:22:12,337: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\984f8177e4e24cb287e1dbd543f5b8be\artifacts\model\data\model]
[2024-01-12 13:22:12,337: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\984f8177e4e24cb287e1dbd543f5b8be\artifacts\model\data\model\assets]
[2024-01-12 13:22:12,345: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\984f8177e4e24cb287e1dbd543f5b8be\artifacts\model\data\model\variables]
[2024-01-12 13:22:12,429: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-12 13:26:39,170: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 13:26:39,170: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 13:26:39,170: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 13:26:39,170: INFO: common: created directory at: artifacts]
[2024-01-12 13:26:39,170: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 13:26:39,170: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:26:40,006: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3 into file artifacts/data_ingestion/data.zip]
[2024-01-12 13:26:40,007: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 14:03:21,213: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 14:03:21,229: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:03:21,240: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:03:21,240: INFO: common: created directory at: artifacts]
[2024-01-12 14:03:21,240: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 14:03:21,240: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:03:23,198: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:03:23,198: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 14:05:19,141: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 14:05:19,141: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:05:19,141: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:05:19,141: INFO: common: created directory at: artifacts]
[2024-01-12 14:05:19,141: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 14:05:19,141: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:05:21,086: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:05:21,102: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 14:06:28,569: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 14:06:28,584: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:06:28,584: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:06:28,584: INFO: common: created directory at: artifacts]
[2024-01-12 14:06:28,584: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 14:06:28,584: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:06:29,404: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:06:29,404: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-12 14:08:01,675: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 14:08:01,690: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:08:01,690: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:08:01,690: INFO: common: created directory at: artifacts]
[2024-01-12 14:08:01,690: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 14:08:01,697: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:08:02,526: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:08:03,259: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-12 14:08:03,259: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-12 14:08:03,259: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:08:03,275: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:08:03,275: INFO: common: created directory at: artifacts]
[2024-01-12 14:08:03,277: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-12 14:08:03,736: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-12 14:08:04,616: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-12 14:08:04,617: INFO: main: >>>>>>> stage Training started]
[2024-01-12 14:08:04,620: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:08:04,622: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:08:04,623: INFO: common: created directory at: artifacts]
[2024-01-12 14:08:04,623: INFO: common: created directory at: artifacts\training]
[2024-01-12 14:09:46,786: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-12 14:09:46,786: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-12 14:09:46,802: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:09:46,802: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:09:46,802: INFO: common: created directory at: artifacts]
[2024-01-12 14:10:05,528: INFO: common: json file saved at: scores.json]
[2024-01-12 14:10:05,528: INFO: common: json file saved at: scores.json]
[2024-01-12 14:10:07,072: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:10:07,721: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp_4nfbgnk\model\data\model\assets]
[2024-01-12 14:10:17,216: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\ef12e9c8bc1e4d57b0046f82346128b8\artifacts\model\data]
[2024-01-12 14:10:17,216: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\ef12e9c8bc1e4d57b0046f82346128b8\artifacts\model\data\model]
[2024-01-12 14:10:17,216: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\ef12e9c8bc1e4d57b0046f82346128b8\artifacts\model\data\model\assets]
[2024-01-12 14:10:17,216: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\ef12e9c8bc1e4d57b0046f82346128b8\artifacts\model\data\model\variables]
[2024-01-12 14:10:17,302: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-12 14:11:56,187: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:11:56,187: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:11:56,187: INFO: common: created directory at: artifacts]
[2024-01-12 14:11:56,187: INFO: common: created directory at: artifacts\training]
[2024-01-12 14:22:53,090: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:22:53,093: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:22:53,095: INFO: common: created directory at: artifacts]
[2024-01-12 14:25:08,061: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:25:08,064: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:25:08,066: INFO: common: created directory at: artifacts]
[2024-01-12 14:25:25,394: INFO: common: json file saved at: scores.json]
[2024-01-12 14:25:28,829: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:25:29,530: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpfwyaotmp\model\data\model\assets]
[2024-01-12 14:33:47,590: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-12 14:33:47,590: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:33:47,605: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:33:47,605: INFO: common: created directory at: artifacts]
[2024-01-12 14:33:47,605: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-12 14:33:47,605: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:33:48,427: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-12 14:33:49,033: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-12 14:33:49,034: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-12 14:33:49,036: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:33:49,039: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:33:49,039: INFO: common: created directory at: artifacts]
[2024-01-12 14:33:49,039: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-12 14:33:49,492: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-12 14:33:50,805: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-12 14:33:50,805: INFO: main: >>>>>>> stage Training started]
[2024-01-12 14:33:50,820: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:33:50,826: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:33:50,827: INFO: common: created directory at: artifacts]
[2024-01-12 14:33:50,828: INFO: common: created directory at: artifacts\training]
[2024-01-12 14:35:37,368: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-12 14:35:37,368: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-12 14:35:37,368: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:35:37,368: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:35:37,368: INFO: common: created directory at: artifacts]
[2024-01-12 14:35:55,858: INFO: common: json file saved at: scores.json]
[2024-01-12 14:35:55,859: INFO: common: json file saved at: scores.json]
[2024-01-12 14:35:57,480: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:35:58,121: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp21i4nn6b\model\data\model\assets]
[2024-01-12 14:36:07,125: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\290f3cb8b2784711a30ea8b1ef794ecc\artifacts\model\data]
[2024-01-12 14:36:07,136: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\290f3cb8b2784711a30ea8b1ef794ecc\artifacts\model\data\model]
[2024-01-12 14:36:07,137: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\290f3cb8b2784711a30ea8b1ef794ecc\artifacts\model\data\model\assets]
[2024-01-12 14:36:07,142: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\290f3cb8b2784711a30ea8b1ef794ecc\artifacts\model\data\model\variables]
[2024-01-12 14:36:07,221: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-12 14:37:51,971: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:37:51,977: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:37:51,978: INFO: common: created directory at: artifacts]
[2024-01-12 14:38:10,182: INFO: common: json file saved at: scores.json]
[2024-01-12 14:38:13,316: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:38:13,966: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmplc_jmajt\model\data\model\assets]
[2024-01-12 14:43:34,319: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:43:34,322: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:43:34,323: INFO: common: created directory at: artifacts]
[2024-01-12 14:43:51,178: INFO: common: json file saved at: scores.json]
[2024-01-12 14:43:53,695: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:43:54,266: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp2dwwc72r\model\data\model\assets]
[2024-01-12 14:47:16,379: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:47:16,383: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:47:16,384: INFO: common: created directory at: artifacts]
[2024-01-12 14:47:34,549: INFO: common: json file saved at: scores.json]
[2024-01-12 14:47:37,215: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:47:37,847: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpxa_wl8wa\model\data\model\assets]
[2024-01-12 14:53:28,265: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-12 14:53:28,272: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-12 14:53:28,272: INFO: common: created directory at: artifacts]
[2024-01-12 14:53:49,502: INFO: common: json file saved at: scores.json]
[2024-01-12 14:53:54,342: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-12 14:53:55,208: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp1b4inzss\model\data\model\assets]
[2024-01-13 09:50:44,079: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 09:50:44,425: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 09:50:44,442: INFO: common: created directory at: artifacts]
[2024-01-13 09:51:04,236: INFO: common: json file saved at: scores.json]
[2024-01-13 09:51:08,065: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-13 09:51:08,732: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp8fuz9jnd\model\data\model\assets]
[2024-01-13 09:57:02,664: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-13 09:57:02,664: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 09:57:02,679: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 09:57:02,679: INFO: common: created directory at: artifacts]
[2024-01-13 09:57:02,679: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-13 09:57:02,679: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-13 09:57:03,499: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-13 09:57:04,592: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-13 09:57:04,592: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-13 09:57:04,592: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 09:57:04,606: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 09:57:04,608: INFO: common: created directory at: artifacts]
[2024-01-13 09:57:04,608: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-13 09:57:05,079: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-13 09:57:06,088: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-13 09:57:06,088: INFO: main: >>>>>>> stage Training started]
[2024-01-13 09:57:06,088: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 09:57:06,088: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 09:57:06,104: INFO: common: created directory at: artifacts]
[2024-01-13 09:57:06,104: INFO: common: created directory at: artifacts\training]
[2024-01-13 09:58:48,294: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-13 09:58:48,294: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-13 09:58:48,294: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 09:58:48,294: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 09:58:48,294: INFO: common: created directory at: artifacts]
[2024-01-13 09:59:05,956: INFO: common: json file saved at: scores.json]
[2024-01-13 09:59:05,956: INFO: common: json file saved at: scores.json]
[2024-01-13 09:59:07,601: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-13 09:59:08,218: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpvkxipim2\model\data\model\assets]
[2024-01-13 09:59:15,465: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\74b874f284544d5e8c5e5475030b5e80\artifacts\model\data]
[2024-01-13 09:59:15,465: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\74b874f284544d5e8c5e5475030b5e80\artifacts\model\data\model]
[2024-01-13 09:59:15,465: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\74b874f284544d5e8c5e5475030b5e80\artifacts\model\data\model\assets]
[2024-01-13 09:59:15,486: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\74b874f284544d5e8c5e5475030b5e80\artifacts\model\data\model\variables]
[2024-01-13 09:59:15,596: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-13 12:00:07,599: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-13 12:00:07,748: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:00:07,748: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:00:07,748: INFO: common: created directory at: artifacts]
[2024-01-13 12:00:07,761: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-13 12:00:07,764: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-13 12:00:08,728: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-13 12:00:09,322: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-13 12:00:09,322: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-13 12:00:09,331: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:00:09,331: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:00:09,331: INFO: common: created directory at: artifacts]
[2024-01-13 12:00:09,331: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-13 12:00:10,033: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-13 12:00:10,996: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-13 12:00:10,996: INFO: main: >>>>>>> stage Training started]
[2024-01-13 12:00:11,005: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:00:11,022: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:00:11,025: INFO: common: created directory at: artifacts]
[2024-01-13 12:00:11,026: INFO: common: created directory at: artifacts\training]
[2024-01-13 12:02:13,674: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-13 12:02:13,674: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-13 12:02:13,680: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:02:13,685: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:02:13,686: INFO: common: created directory at: artifacts]
[2024-01-13 12:02:33,829: INFO: common: json file saved at: scores.json]
[2024-01-13 12:02:33,829: INFO: common: json file saved at: scores.json]
[2024-01-13 12:02:37,993: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-13 12:02:38,916: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpdwqbe27b\model\data\model\assets]
[2024-01-13 12:03:09,243: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-13 12:03:58,746: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-13 12:03:58,764: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:03:58,767: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:03:58,767: INFO: common: created directory at: artifacts]
[2024-01-13 12:03:58,769: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-13 12:03:58,769: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-13 12:03:59,743: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-13 12:04:00,318: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-13 12:04:00,318: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-13 12:04:00,321: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:04:00,324: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:04:00,324: INFO: common: created directory at: artifacts]
[2024-01-13 12:04:00,324: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-13 12:04:00,842: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-13 12:04:01,633: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-13 12:04:01,633: INFO: main: >>>>>>> stage Training started]
[2024-01-13 12:04:01,649: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:04:01,657: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:04:01,660: INFO: common: created directory at: artifacts]
[2024-01-13 12:04:01,663: INFO: common: created directory at: artifacts\training]
[2024-01-13 12:07:03,751: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-13 12:07:03,751: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-13 12:07:03,751: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-13 12:07:03,766: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-13 12:07:03,767: INFO: common: created directory at: artifacts]
[2024-01-13 12:07:23,560: INFO: common: json file saved at: scores.json]
[2024-01-13 12:07:23,560: INFO: common: json file saved at: scores.json]
[2024-01-13 12:07:27,435: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-13 12:07:28,156: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpnxt8fr8y\model\data\model\assets]
[2024-01-13 12:07:57,293: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-19 11:36:06,504: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-19 11:36:06,536: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 11:36:06,562: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 11:36:06,575: INFO: common: created directory at: artifacts]
[2024-01-19 11:36:06,576: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-19 11:36:06,578: INFO: dtat_ingestion: Downloading data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-19 11:36:07,396: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/uc?/export=download&id=1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9 into file artifacts/data_ingestion/data.zip]
[2024-01-19 11:36:07,399: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-19 11:47:19,753: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-19 11:47:19,768: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 11:47:19,768: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 11:47:19,768: INFO: common: created directory at: artifacts]
[2024-01-19 11:47:19,768: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-19 11:47:19,768: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-19 11:47:21,646: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-19 11:47:21,661: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 12, in <module>
    data_ingestion.main()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\components\dtat_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-01-19 12:00:52,237: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-01-19 12:00:52,237: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:00:52,237: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:00:52,237: INFO: common: created directory at: artifacts]
[2024-01-19 12:00:52,237: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-19 12:00:52,237: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-19 12:00:54,137: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-19 12:00:55,221: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-19 12:00:55,221: INFO: main: >>>>>>> stage Prepare base model started]
[2024-01-19 12:00:55,221: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:00:55,221: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:00:55,237: INFO: common: created directory at: artifacts]
[2024-01-19 12:00:55,237: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-19 12:00:55,827: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-19 12:00:56,491: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-19 12:00:56,491: INFO: main: >>>>>>> stage Training started]
[2024-01-19 12:00:56,506: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:00:56,508: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:00:56,508: INFO: common: created directory at: artifacts]
[2024-01-19 12:00:56,508: INFO: common: created directory at: artifacts\training]
[2024-01-19 12:03:45,088: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-19 12:03:45,091: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-01-19 12:03:45,100: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:03:45,104: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:03:45,105: INFO: common: created directory at: artifacts]
[2024-01-19 12:04:04,740: INFO: common: json file saved at: scores.json]
[2024-01-19 12:04:04,740: INFO: common: json file saved at: scores.json]
[2024-01-19 12:04:06,412: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-19 12:04:07,153: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp6cu3ijj_\model\data\model\assets]
[2024-01-19 12:04:15,438: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\1b1d441ce2154410bf055dbdc7054a5f\artifacts\model\data]
[2024-01-19 12:04:15,438: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\1b1d441ce2154410bf055dbdc7054a5f\artifacts\model\data\model]
[2024-01-19 12:04:15,438: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\1b1d441ce2154410bf055dbdc7054a5f\artifacts\model\data\model\assets]
[2024-01-19 12:04:15,452: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\1b1d441ce2154410bf055dbdc7054a5f\artifacts\model\data\model\variables]
[2024-01-19 12:04:15,554: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-19 12:05:31,081: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started]
[2024-01-19 12:05:31,081: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:05:31,081: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:05:31,081: INFO: common: created directory at: artifacts]
[2024-01-19 12:05:31,081: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-19 12:05:31,091: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-19 12:05:33,111: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-01-19 12:05:33,852: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-01-19 12:05:44,066: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started]
[2024-01-19 12:05:44,066: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:05:44,066: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:05:44,066: INFO: common: created directory at: artifacts]
[2024-01-19 12:05:44,066: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-01-19 12:05:44,487: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-01-19 12:05:44,915: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-01-19 12:05:55,125: INFO: stage_03_model_training: >>>>>>> stage Training started]
[2024-01-19 12:05:55,125: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:05:55,135: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:05:55,135: INFO: common: created directory at: artifacts]
[2024-01-19 12:05:55,135: INFO: common: created directory at: artifacts\training]
[2024-01-19 12:08:44,913: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-19 12:15:28,525: INFO: stage_03_model_training: >>>>>>> stage Training started]
[2024-01-19 12:15:28,525: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:15:28,525: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:15:28,525: INFO: common: created directory at: artifacts]
[2024-01-19 12:15:28,525: INFO: common: created directory at: artifacts\training]
[2024-01-19 12:16:18,247: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-01-19 12:17:51,818: INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage started]
[2024-01-19 12:17:51,818: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-19 12:17:51,818: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-19 12:17:51,818: INFO: common: created directory at: artifacts]
[2024-01-19 12:18:08,082: INFO: common: json file saved at: scores.json]
[2024-01-19 12:18:08,082: INFO: common: json file saved at: scores.json]
[2024-01-19 12:18:09,447: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-01-19 12:18:10,103: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpjf52oeg7\model\data\model\assets]
[2024-01-19 12:18:17,775: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\bb33c33b95f040aeab40212070abab88\artifacts\model\data]
[2024-01-19 12:18:17,775: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\bb33c33b95f040aeab40212070abab88\artifacts\model\data\model]
[2024-01-19 12:18:17,775: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\bb33c33b95f040aeab40212070abab88\artifacts\model\data\model\assets]
[2024-01-19 12:18:17,784: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\bb33c33b95f040aeab40212070abab88\artifacts\model\data\model\variables]
[2024-01-19 12:18:17,964: INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-01-19 13:18:21,315: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 13:18:21,318: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 13:18:29,471: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:18:29] "GET / HTTP/1.1" 200 -]
[2024-01-19 13:19:16,962: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-01-19 13:19:16,968: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:19:16] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-01-19 13:25:44,049: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 13:25:44,049: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 13:25:59,005: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:25:59] "GET / HTTP/1.1" 200 -]
[2024-01-19 13:26:11,133: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-01-19 13:26:11,133: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:26:11] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-01-19 13:34:05,583: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 13:34:05,583: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 13:34:09,479: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:34:09] "GET / HTTP/1.1" 200 -]
[2024-01-19 13:34:16,198: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("training", "model.h5"))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at training\model.h5
[2024-01-19 13:34:16,198: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:34:16] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-01-19 13:39:21,213: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 13:39:21,213: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 13:39:26,198: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:39:26] "GET / HTTP/1.1" 200 -]
[2024-01-19 13:39:33,685: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "e:\kidney-disease-classification-deep-learning-project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("E:\Kidney-Disease-Classification-Deep-Learning-Project\artifacts\training", "model.h5"))
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\KAPIL\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at E:\Kidney-Disease-Classification-Deep-Learning-Projectrtifacts	raining\model.h5
[2024-01-19 13:39:33,699: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:39:33] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-01-19 13:41:07,050: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 13:41:07,050: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 13:41:21,404: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:41:21] "GET / HTTP/1.1" 200 -]
[2024-01-19 13:41:28,811: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:41:28] "POST /predict HTTP/1.1" 200 -]
[2024-01-19 13:41:51,470: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:41:51] "POST /predict HTTP/1.1" 200 -]
[2024-01-19 13:41:59,422: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 13:41:59] "POST /predict HTTP/1.1" 200 -]
[2024-01-19 16:19:46,747: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 16:19:46,851: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 16:19:50,293: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 16:19:50] "GET / HTTP/1.1" 200 -]
[2024-01-19 16:20:03,223: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 16:20:03] "POST /predict HTTP/1.1" 200 -]
[2024-01-19 16:20:12,847: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 16:20:12] "POST /predict HTTP/1.1" 200 -]
[2024-01-19 16:30:51,297: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.173:8080]
[2024-01-19 16:30:51,299: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-01-19 16:30:52,975: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 16:30:52] "GET / HTTP/1.1" 200 -]
[2024-01-19 16:31:07,129: INFO: _internal: 127.0.0.1 - - [19/Jan/2024 16:31:07] "POST /predict HTTP/1.1" 200 -]
[2024-04-07 15:36:45,676: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-07 15:36:45,783: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-07 15:36:45,785: INFO: common: created directory at: artifacts]
[2024-04-07 15:36:45,785: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-07 15:36:45,785: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 15:37:53,531: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-07 15:37:53,538: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-07 15:37:53,538: INFO: common: created directory at: artifacts]
[2024-04-07 15:37:53,542: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-07 15:37:53,546: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 15:38:53,972: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 17:31:49,770: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-07 17:31:49,870: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-07 17:31:49,870: INFO: common: created directory at: artifacts]
[2024-04-07 17:31:49,870: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-07 17:31:49,870: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 17:32:40,503: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1V0n5SMeUwC5E6fnLSoUpNFUOCIzpHoy9/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 18:12:29,879: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-07 18:12:29,879: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-07 18:12:29,879: INFO: common: created directory at: artifacts]
[2024-04-07 18:12:29,879: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-07 18:12:29,879: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 18:13:35,249: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 19:15:41,329: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-07 19:15:41,335: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-07 19:15:41,335: INFO: common: created directory at: artifacts]
[2024-04-07 19:15:41,339: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-07 19:15:41,339: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 19:16:53,683: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-07 19:16:53,688: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-07 19:16:53,690: INFO: common: created directory at: artifacts]
[2024-04-07 19:16:53,693: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-07 19:16:53,696: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-07 19:17:41,473: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-08 09:57:06,534: INFO: main1: >>>>>>> stage Data Ingestion stage started]
[2024-04-08 09:57:06,549: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-08 09:57:06,549: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-08 09:57:06,549: INFO: common: created directory at: artifacts]
[2024-04-08 09:57:06,549: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-08 09:57:06,549: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-08 09:57:30,178: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-08 09:57:30,718: INFO: main1: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-08 16:15:11,686: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-08 16:15:11,701: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-08 16:15:11,701: INFO: common: created directory at: artifacts]
[2024-04-08 16:15:11,701: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-08 16:15:12,518: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 15:24:49,056: INFO: main1: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 15:24:49,071: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 15:24:49,101: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 15:24:49,103: INFO: common: created directory at: artifacts]
[2024-04-09 15:24:49,106: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 15:24:49,106: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 15:25:31,547: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 15:25:32,775: INFO: main1: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 15:25:32,775: INFO: main1: >>>>>>> stage Prepare base model started]
[2024-04-09 15:25:32,792: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 15:25:32,802: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 15:25:32,807: INFO: common: created directory at: artifacts]
[2024-04-09 15:25:32,807: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 15:25:33,639: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 15:25:35,331: INFO: main1: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 16:13:02,029: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 16:13:02,037: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 16:13:02,039: INFO: common: created directory at: artifacts]
[2024-04-09 16:13:02,041: INFO: common: created directory at: artifacts\training]
[2024-04-09 17:07:16,579: INFO: main1: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 17:07:16,582: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 17:07:16,583: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 17:07:16,584: INFO: common: created directory at: artifacts]
[2024-04-09 17:07:16,584: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 17:07:16,585: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 17:07:56,594: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 17:07:57,017: INFO: main1: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 17:07:57,017: INFO: main1: >>>>>>> stage Prepare base model started]
[2024-04-09 17:07:57,017: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 17:07:57,017: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 17:07:57,017: INFO: common: created directory at: artifacts]
[2024-04-09 17:07:57,017: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 17:07:57,471: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 17:07:58,998: INFO: main1: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 17:07:58,998: INFO: main1: >>>>>>> stage Training started]
[2024-04-09 17:07:58,998: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 17:07:58,998: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 17:07:58,998: INFO: common: created directory at: artifacts]
[2024-04-09 17:07:58,998: INFO: common: created directory at: artifacts\training]
[2024-04-09 17:08:52,947: INFO: main1: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-09 18:02:25,525: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 18:02:25,525: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 18:02:25,541: INFO: common: created directory at: artifacts]
[2024-04-09 18:02:44,115: INFO: common: json file saved at: scores.json]
[2024-04-09 18:02:48,360: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-09 18:02:49,050: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmps1kpg84v\model\data\model\assets]
[2024-04-09 18:32:23,660: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 18:32:23,661: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 18:32:23,679: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 18:32:23,679: INFO: common: created directory at: artifacts]
[2024-04-09 18:32:23,681: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 18:32:23,681: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 18:32:54,693: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 18:32:55,313: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 18:32:55,315: INFO: main: >>>>>>> stage Prepare base model started]
[2024-04-09 18:32:55,318: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 18:32:55,318: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 18:32:55,318: INFO: common: created directory at: artifacts]
[2024-04-09 18:32:55,321: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 18:32:55,852: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 18:32:57,300: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 18:32:57,300: INFO: main: >>>>>>> stage Training started]
[2024-04-09 18:32:57,305: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 18:32:57,322: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 18:32:57,323: INFO: common: created directory at: artifacts]
[2024-04-09 18:32:57,325: INFO: common: created directory at: artifacts\training]
[2024-04-09 18:35:05,902: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-09 18:35:05,902: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-04-09 18:35:05,915: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 18:35:05,919: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 18:35:05,919: INFO: common: created directory at: artifacts]
[2024-04-09 18:35:27,504: INFO: common: json file saved at: scores.json]
[2024-04-09 18:35:27,504: INFO: common: json file saved at: scores.json]
[2024-04-09 18:35:27,504: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:10:43,841: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 19:10:43,847: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:10:43,851: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:10:43,851: INFO: common: created directory at: artifacts]
[2024-04-09 19:10:43,853: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 19:10:43,853: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:11:45,412: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:11:46,024: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:11:46,024: INFO: main: >>>>>>> stage Prepare base model started]
[2024-04-09 19:11:46,024: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:11:46,035: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:11:46,035: INFO: common: created directory at: artifacts]
[2024-04-09 19:11:46,035: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 19:11:46,618: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 19:11:48,440: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 19:11:48,455: INFO: main: >>>>>>> stage Training started]
[2024-04-09 19:11:48,456: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:11:48,467: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:11:48,468: INFO: common: created directory at: artifacts]
[2024-04-09 19:11:48,470: INFO: common: created directory at: artifacts\training]
[2024-04-09 19:13:56,577: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-09 19:13:56,592: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-04-09 19:13:56,594: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:13:56,594: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:13:56,594: INFO: common: created directory at: artifacts]
[2024-04-09 19:14:17,055: INFO: common: json file saved at: scores.json]
[2024-04-09 19:14:17,055: INFO: common: json file saved at: scores.json]
[2024-04-09 19:14:17,055: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:14:51,094: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 19:14:51,097: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:14:51,100: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:14:51,100: INFO: common: created directory at: artifacts]
[2024-04-09 19:14:51,100: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 19:14:51,102: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:15:40,976: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:15:41,592: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:15:41,593: INFO: main: >>>>>>> stage Prepare base model started]
[2024-04-09 19:15:41,595: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:15:41,597: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:15:41,597: INFO: common: created directory at: artifacts]
[2024-04-09 19:15:41,600: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 19:15:42,254: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 19:15:43,961: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 19:15:43,962: INFO: main: >>>>>>> stage Training started]
[2024-04-09 19:15:43,967: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:15:43,971: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:15:43,972: INFO: common: created directory at: artifacts]
[2024-04-09 19:15:43,973: INFO: common: created directory at: artifacts\training]
[2024-04-09 19:17:52,762: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-09 19:17:52,762: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-04-09 19:17:52,765: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:17:52,781: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:17:52,782: INFO: common: created directory at: artifacts]
[2024-04-09 19:18:13,805: INFO: common: json file saved at: scores.json]
[2024-04-09 19:18:13,805: INFO: common: json file saved at: scores.json]
[2024-04-09 19:18:13,805: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:31:27,072: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 19:31:27,078: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:31:27,082: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:31:27,082: INFO: common: created directory at: artifacts]
[2024-04-09 19:31:27,082: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 19:31:27,088: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:32:41,888: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:32:42,458: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:32:42,458: INFO: main: >>>>>>> stage Prepare base model started]
[2024-04-09 19:32:42,470: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:32:42,473: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:32:42,474: INFO: common: created directory at: artifacts]
[2024-04-09 19:32:42,474: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 19:32:43,090: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 19:32:44,357: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 19:32:44,357: INFO: main: >>>>>>> stage Training started]
[2024-04-09 19:32:44,359: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:32:44,365: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:32:44,366: INFO: common: created directory at: artifacts]
[2024-04-09 19:32:44,366: INFO: common: created directory at: artifacts\training]
[2024-04-09 19:34:52,683: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-09 19:34:52,683: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-04-09 19:34:52,693: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:34:52,707: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:34:52,709: INFO: common: created directory at: artifacts]
[2024-04-09 19:35:13,688: INFO: common: json file saved at: scores.json]
[2024-04-09 19:35:13,688: INFO: common: json file saved at: scores.json]
[2024-04-09 19:35:15,914: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-09 19:35:16,697: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpiljzaod8\model\data\model\assets]
[2024-04-09 19:35:26,525: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\8f6af8365f434d3da9f481cf004a309b\artifacts\model\data]
[2024-04-09 19:35:26,528: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\8f6af8365f434d3da9f481cf004a309b\artifacts\model\data\model]
[2024-04-09 19:35:26,535: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\8f6af8365f434d3da9f481cf004a309b\artifacts\model\data\model\assets]
[2024-04-09 19:35:26,550: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\8f6af8365f434d3da9f481cf004a309b\artifacts\model\data\model\variables]
[2024-04-09 19:35:26,663: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:41:08,407: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-04-09 19:41:08,412: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:41:08,412: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:41:08,412: INFO: common: created directory at: artifacts]
[2024-04-09 19:41:08,412: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-09 19:41:08,412: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:42:20,490: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-09 19:42:21,075: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-09 19:42:21,099: INFO: main: >>>>>>> stage Prepare base model started]
[2024-04-09 19:42:21,107: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:42:21,110: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:42:21,112: INFO: common: created directory at: artifacts]
[2024-04-09 19:42:21,112: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-09 19:42:21,686: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-09 19:42:23,111: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-09 19:42:23,111: INFO: main: >>>>>>> stage Training started]
[2024-04-09 19:42:23,127: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:42:23,135: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:42:23,137: INFO: common: created directory at: artifacts]
[2024-04-09 19:42:23,137: INFO: common: created directory at: artifacts\training]
[2024-04-09 19:44:32,590: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-09 19:44:32,598: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-04-09 19:44:32,600: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-09 19:44:32,604: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-09 19:44:32,604: INFO: common: created directory at: artifacts]
[2024-04-09 19:44:53,753: INFO: common: json file saved at: scores.json]
[2024-04-09 19:44:53,754: INFO: common: json file saved at: scores.json]
[2024-04-09 19:44:58,659: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-09 19:44:59,452: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp9h9_o243\model\data\model\assets]
[2024-04-09 19:47:39,875: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-10 15:03:46,253: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started]
[2024-04-10 15:03:46,399: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-10 15:03:46,404: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-10 15:03:46,804: INFO: common: created directory at: artifacts]
[2024-04-10 15:03:46,804: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-10 15:03:46,805: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-10 15:04:10,656: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-10 15:04:11,449: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-10 15:04:29,010: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started]
[2024-04-10 15:04:29,013: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-10 15:04:29,014: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-10 15:04:29,015: INFO: common: created directory at: artifacts]
[2024-04-10 15:04:29,017: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-10 15:04:29,637: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-10 15:04:30,701: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-10 15:04:39,057: INFO: stage_03_model_training: >>>>>>> stage Training started]
[2024-04-10 15:04:39,059: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-10 15:04:39,061: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-10 15:04:39,062: INFO: common: created directory at: artifacts]
[2024-04-10 15:04:39,062: INFO: common: created directory at: artifacts\training]
[2024-04-10 15:06:42,618: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-10 15:06:56,638: INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage started]
[2024-04-10 15:06:56,645: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-10 15:06:56,647: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-10 15:06:56,648: INFO: common: created directory at: artifacts]
[2024-04-10 15:07:17,180: INFO: common: json file saved at: scores.json]
[2024-04-10 15:07:17,182: INFO: common: json file saved at: scores.json]
[2024-04-10 15:07:19,306: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-10 15:07:20,090: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmptmuqg_w6\model\data\model\assets]
[2024-04-10 15:07:30,528: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\715e885d9cba4cae89854c023566c7bb\artifacts\model\data]
[2024-04-10 15:07:30,532: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\715e885d9cba4cae89854c023566c7bb\artifacts\model\data\model]
[2024-04-10 15:07:30,534: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\715e885d9cba4cae89854c023566c7bb\artifacts\model\data\model\assets]
[2024-04-10 15:07:30,541: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\715e885d9cba4cae89854c023566c7bb\artifacts\model\data\model\variables]
[2024-04-10 15:07:30,640: INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-10 17:16:36,101: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.125.65.203:8080]
[2024-04-10 17:16:36,199: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-10 17:16:47,874: INFO: _internal: 127.0.0.1 - - [10/Apr/2024 17:16:47] "GET / HTTP/1.1" 200 -]
[2024-04-10 17:17:22,026: INFO: _internal: 127.0.0.1 - - [10/Apr/2024 17:17:22] "POST /predict HTTP/1.1" 200 -]
[2024-04-10 17:17:37,271: INFO: _internal: 127.0.0.1 - - [10/Apr/2024 17:17:37] "POST /predict HTTP/1.1" 200 -]
[2024-04-10 17:17:40,695: INFO: _internal: 127.0.0.1 - - [10/Apr/2024 17:17:40] "POST /predict HTTP/1.1" 200 -]
[2024-04-10 17:17:52,090: INFO: _internal: 127.0.0.1 - - [10/Apr/2024 17:17:52] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 10:56:45,331: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 10:56:45,331: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 10:56:50,737: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 10:56:50] "GET / HTTP/1.1" 200 -]
[2024-04-11 10:57:00,931: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 10:57:00] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 11:12:17,654: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 11:12:17,654: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 11:12:25,052: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:12:25] "GET / HTTP/1.1" 200 -]
[2024-04-11 11:12:44,944: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:12:44] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 11:36:32,541: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 11:36:32,541: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 11:36:34,520: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:36:34] "GET / HTTP/1.1" 200 -]
[2024-04-11 11:36:34,556: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:36:34] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-04-11 11:36:47,248: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:36:47] "[33mPOST /upload-image HTTP/1.1[0m" 404 -]
[2024-04-11 11:36:47,337: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:36:47] "[33mPOST /upload-image HTTP/1.1[0m" 404 -]
[2024-04-11 11:41:31,984: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 11:41:31,985: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 11:41:36,312: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:41:36] "GET / HTTP/1.1" 200 -]
[2024-04-11 11:48:01,167: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 11:48:01] "[31m[1mGET /predict HTTP/1.1[0m" 405 -]
[2024-04-11 12:27:11,994: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 12:27:11,994: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 12:27:14,834: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:27:14] "GET / HTTP/1.1" 200 -]
[2024-04-11 12:27:25,021: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:27:25] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 12:27:47,735: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:27:47] "[31m[1mGET /predict HTTP/1.1[0m" 405 -]
[2024-04-11 12:43:00,352: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 12:43:00,354: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 12:43:01,926: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:43:01] "GET / HTTP/1.1" 200 -]
[2024-04-11 12:43:10,245: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:43:10] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 12:51:19,047: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 12:51:19,048: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 12:51:35,042: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:51:35] "GET / HTTP/1.1" 200 -]
[2024-04-11 12:51:44,758: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:51:44] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 12:59:36,471: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:59:36] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 12:59:47,985: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 12:59:47] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 13:23:48,399: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 13:23:48,399: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 13:23:50,982: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 13:23:50] "GET / HTTP/1.1" 200 -]
[2024-04-11 13:24:06,606: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 13:24:06] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 13:35:17,268: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.31:8080]
[2024-04-11 13:35:17,269: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 13:35:18,878: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 13:35:18] "GET / HTTP/1.1" 200 -]
[2024-04-11 13:35:33,000: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 13:35:33] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 13:36:22,518: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 13:36:22] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 13:36:33,856: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 13:36:33] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 15:12:10,366: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:80
 * Running on http://192.168.29.31:80]
[2024-04-11 15:12:10,467: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-11 15:12:12,396: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 15:12:12] "GET / HTTP/1.1" 200 -]
[2024-04-11 15:12:12,493: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 15:12:12] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-04-11 15:12:18,726: INFO: _internal: 127.0.0.1 - - [11/Apr/2024 15:12:18] "POST /predict HTTP/1.1" 200 -]
[2024-04-11 15:15:05,271: INFO: main: >>>>>>> stage Data Ingestion stage started]
[2024-04-11 15:15:05,271: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-11 15:15:05,286: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-11 15:15:05,287: INFO: common: created directory at: artifacts]
[2024-04-11 15:15:05,288: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-11 15:15:05,288: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-11 15:15:09,670: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-11 15:15:10,136: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-11 15:15:10,136: INFO: main: >>>>>>> stage Prepare base model started]
[2024-04-11 15:15:10,136: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-11 15:15:10,136: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-11 15:15:10,136: INFO: common: created directory at: artifacts]
[2024-04-11 15:15:10,136: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-11 15:15:10,722: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-11 15:15:11,964: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-11 15:15:11,964: INFO: main: >>>>>>> stage Training started]
[2024-04-11 15:15:11,969: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-11 15:15:11,974: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-11 15:15:11,975: INFO: common: created directory at: artifacts]
[2024-04-11 15:15:11,976: INFO: common: created directory at: artifacts\training]
[2024-04-11 15:17:06,205: INFO: main: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-11 15:17:06,205: INFO: main: >>>>>>> stage Evaluation stage started]
[2024-04-11 15:17:06,205: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-11 15:17:06,205: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-11 15:17:06,205: INFO: common: created directory at: artifacts]
[2024-04-11 15:17:25,989: INFO: common: json file saved at: scores.json]
[2024-04-11 15:17:25,990: INFO: common: json file saved at: scores.json]
[2024-04-11 15:17:27,892: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-11 15:17:28,683: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmpycph3d2x\model\data\model\assets]
[2024-04-11 15:17:37,882: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\246d2606e317407c96cd6ffd92484ac1\artifacts\model\data]
[2024-04-11 15:17:37,884: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\246d2606e317407c96cd6ffd92484ac1\artifacts\model\data\model]
[2024-04-11 15:17:37,885: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\246d2606e317407c96cd6ffd92484ac1\artifacts\model\data\model\assets]
[2024-04-11 15:17:37,887: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\246d2606e317407c96cd6ffd92484ac1\artifacts\model\data\model\variables]
[2024-04-11 15:17:37,978: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
[2024-04-12 08:40:12,785: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started]
[2024-04-12 08:40:12,789: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-12 08:40:12,789: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-12 08:40:12,789: INFO: common: created directory at: artifacts]
[2024-04-12 08:40:12,796: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-12 08:40:12,797: INFO: dtat_ingestion: Downloading data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-12 08:40:36,619: INFO: dtat_ingestion: Downloaded data from https://drive.google.com/file/d/1h_vUZNurCue1KF3I4XStiB9v3GLLQGpc/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-12 08:40:37,606: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<<<< 

x=========x]
[2024-04-12 08:41:03,651: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started]
[2024-04-12 08:41:03,657: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-12 08:41:03,660: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-12 08:41:03,660: INFO: common: created directory at: artifacts]
[2024-04-12 08:41:03,663: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-12 08:41:04,423: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-12 08:41:05,233: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<<<< 

x=========x]
[2024-04-12 08:41:15,053: INFO: stage_03_model_training: >>>>>>> stage Training started]
[2024-04-12 08:41:15,056: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-12 08:41:15,060: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-12 08:41:15,060: INFO: common: created directory at: artifacts]
[2024-04-12 08:41:15,062: INFO: common: created directory at: artifacts\training]
[2024-04-12 08:44:13,986: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<<<< 

x=========x]
[2024-04-12 08:44:24,786: INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage started]
[2024-04-12 08:44:24,789: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-12 08:44:24,793: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-12 08:44:24,794: INFO: common: created directory at: artifacts]
[2024-04-12 08:44:43,810: INFO: common: json file saved at: scores.json]
[2024-04-12 08:44:43,810: INFO: common: json file saved at: scores.json]
[2024-04-12 08:44:45,685: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-12 08:44:46,476: INFO: builder_impl: Assets written to: C:\Users\KAPIL\AppData\Local\Temp\tmp3nw_5y92\model\data\model\assets]
[2024-04-12 08:45:00,736: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\2cd35b259f0e4b5ba3c172c7c495f1eb\artifacts\model\data]
[2024-04-12 08:45:00,752: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\2cd35b259f0e4b5ba3c172c7c495f1eb\artifacts\model\data\model]
[2024-04-12 08:45:00,752: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\2cd35b259f0e4b5ba3c172c7c495f1eb\artifacts\model\data\model\assets]
[2024-04-12 08:45:00,766: INFO: dir_util: creating E:\Kidney-Disease-Classification-Deep-Learning-Project\mlruns\0\2cd35b259f0e4b5ba3c172c7c495f1eb\artifacts\model\data\model\variables]
[2024-04-12 08:45:00,863: INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage completed <<<<<<<<< 

x=========x]
